src/app/material-docs-app.scss

```diff
@@ -26,6 +26,28 @@ material-docs-app > guide-viewer {
   overflow-y: auto;
 }
 
+#skip-link-wrapper {
+  z-index: 10;
+  position: fixed;
```

Comment:
Should this be moved inside the `app-navbar`? That would probably make it so we don't need the `position: fixed` and `z-index`, since the app bar already has fixed positioning